### ISSUE TITLE
Option to disable update check

### ISSUE THREAD

**Issue by soredake** [abgox/PSCompletions#5]
I have a script that updates PowerShell modules once a week, I don't need this update check:
![image](https://github.com/abgox/PSCompletions/assets/5204968/820147c0-d083-446b-be30-ffbefdd69401)

**Comment by abgox** [comment]
- Unfortunately,it's necessary, and I won't add a configuration to disable it either.
- This is an update of the module and must be kept up-to-date, as all the completion in the repo is for the latest version service.
- If the version is not the latest, Adding, updating, and removing completion may trigger unexpected incompatibility errors
-  Of course, frequent updates are a problem, but this proves that there are some areas that need to be modified, and once stable, such updates will become less and less frequent

- In other words, it's updates are not complex, you just need to press `Enter` and wait it.

**Comment by soredake** [comment]
> * Unfortunately,it's necessary

For me, it's not, I don't use these completions this often, not having a choice to disable this isn't good.

**Comment by soredake** [comment]
And needing to press any key TWICE to skip this updater is ridiculous:
![image](https://github.com/abgox/PSCompletions/assets/5204968/354f4829-8aee-45fb-b707-2ad7948002af)

**Comment by abgox** [comment]
- As mentioned in my previous update logs, there is no issue with the module.
- This should be due to other modules you are using affecting it.

**Comment by soredake** [comment]
> * Of course, frequent updates are a problem

It's not a problem when you are not annoyed by this update check that you don't really need.

**Comment by abgox** [comment]
Just press `Enter`, it won't take up much of your time.
When you are the latest version, the update check will not appear.

**Comment by soredake** [comment]
> Just press `Enter`, it won't take up much of your time.

I don't want to, I set up a weekly update script not to ever update anything manually.

And most importantly, I don't think this module is important enough to block you from using shell until you do something, nothing bad (or anything at all probably) will happen if i just not update module or completions.

**Comment by abgox** [comment]
If you are unwilling to do such a simple thing(Only press `Enter`), why not uninstall it directly?
As you said, this module  isn't important.

**Comment by soredake** [comment]
> this module isn't important.

That's not what i said, i said `I don't think this module is important enough to block you from using shell until you do something`, i do want to use this module, just without this update check.

**Comment by abgox** [comment]
- Perhaps I can add this option and describe it as a warning option.
- But If there are no other content updates, it will not have the next version.
- I won't update the version just only for this option.

**Comment by abgox** [comment]
In the new version, this option has been added.

**Comment by soredake** [comment]
@abgox thank you!
